cpp/log/cert.cc

```diff
@@ -233,8 +262,8 @@ string Cert::PrintTime(ASN1_TIME* when) {
 }
 
 
-Cert::Status Cert::IsIdenticalTo(const Cert& other) const {
-  return X509_cmp(x509_, other.x509_) == 0 ? TRUE : FALSE;
+StatusOr<bool> Cert::IsIdenticalTo(const Cert& other) const {
+  return X509_cmp(x509_, other.x509_) == 0 ? true : false;
```

Comment:
Also, this could be a proper `bool` instead of a `StatusOr<bool>`, could make other call sites simpler?